pandas/core/arrays/numpy_.py

```diff
@@ -172,9 +171,13 @@ def __init__(self, values: Union[np.ndarray, "PandasArray"], copy: bool = False)
         if copy:
             values = values.copy()
 
-        self._ndarray = values
+        self._data = values
         self._dtype = PandasDtype(values.dtype)
 
+    @property
+    def _ndarray(self) -> np.ndarray:
+        return self._data
```

Comment:
IIRC this has perf implications that add up

Reply:
they are already properties in Categorial and DateTimeLike. do you recall where the perf imp could be.

Reply:
not off the top of my head; it shouldnt be that big of a deal

i had expected mypy to understand an attribute set in `__init__` or constant at the class level as being equivalent to a property